upgrading/topics/keycloak/changes-18_0_0.adoc

```diff
@@ -0,0 +1,20 @@
+= Step-up authentication
+
+Step-up authentication is new feature added. There is `acr` client scope, which contains protocol mapper, which is supposed to add `acr`
+claim in the token. So the `acr` claim is not added automatically now as it was before this version, but it is added with the usage
+of this client scope and protocol mapper.
+
+The client scope is added as realm "default" client scope and hence will be added to all newly created clients. But for performance reasons,
+the client scope is not automatically added to all existing clients during migration. So clients won't have `acr` claim by default after
+the migration. Few possibilities to handle this:
+
+- If you don't plan to use step-up authentication feature, but you rely on the `acr` claim in the token, you can disable `step_up_authentication`
```

Comment:
It might be good that L11-L12 be good sentence.

Reply:
I kept single sentence. Same as above.

Reply:
Sorry. What I wanted to say is to remove linefeeds among L11-L12. Not write as them one sentence.

Reply:
Replied in the comment https://github.com/keycloak/keycloak-documentation/pull/1428#discussion_r824468705 . We can keep the conversation there related to line feeds.

Reply:
Yes, I agree.